lib/currency.rb

```diff
@@ -0,0 +1,29 @@
+module Currency
+  extend self
+  
+  def args(*currency)
+    currency = currency.flatten.compact
```

Comment:
If you want to simplify all these splatting, make `args` to accept `Array` instance as first argument.